osm2gtfs/creators/fenix/trips_creator_fenix.py

```diff
@@ -135,13 +141,15 @@ def add_route(self, feed, route, horarios, operacoes):
         self.add_trips_by_day(feed, line, self.service_sunday, route, sunday, SUNDAY)
 
     def add_trips_by_day(self, feed, line, service, route, horarios, day):
+
         # check if we even have service
         if horarios is None or len(horarios) == 0:
             return
 
-        if isinstance(route, RouteMaster):
+        if isinstance(route, Line):
             # recurse into "Ida" and "Volta" routes
-            for sub_route in route.routes.values():
+            for sub_route in route.get_itineraries():
+                sub_route.duration = route.duration
```

Comment:
Why did you add this?

Reply:
Because before the duration was also stored on the itinerary object. In other words, to not make this pull request bigger than it is... 🤣

Reply:
Can you explain this a bit more please?

Reply:
I just checked the code. And I have to excuse, after weeks or months I can't remember it clearly anymore. I do record, that I added when bugfixing Fenix. If you take it away now, it throws some errors, so this should give you the explanation.

>   File "/home/user/Projects/osm2gtfs/osm2gtfs-code/osm2gtfs/creators/fenix/trips_creator_fenix.py", line 198, in add_trips_by_day
>    end_sec = start_sec + route.duration.seconds * factor
> AttributeError: 'NoneType' object has no attribute 'seconds'

It was the shortest and best fix I came up, without changing too much in the Fenix implementation in this PR.